src/client.cpp

```diff
@@ -821,9 +823,12 @@ void Client::initLocalMapSaving(const Address &address,
 
 	fs::CreateAllDirs(world_path);
 
-	m_localdb = new Database_SQLite3(world_path);
+	Settings conf;
+	conf.set("sqlite_journal_mode", g_settings->get("sqlite_journal_mode"));
+	m_localdb = new Database_SQLite3(world_path, conf);
```

Comment:
This will change the journal mode on existing local maps.  Preferably, it would default to whatever the map was currently using.